### PR TITLE
Doc 737 removing duplication

### PR DESCRIPTION
**Pull request by JungaBunga**

### CODE REVIEW

File: app/constants/report-types.js [hunk 1]
```diff
@@ -2,8 +2,7 @@ const getReportTypes = () => ({
   'Payment request statuses': 'payment-requests',
   'Combined transaction report': 'transaction-summary',
   'Suppressed payment requests': 'suppressed-payments',
-  'AP Listing Report': 'ap-listing',
-  'AR Listing Report': 'ar-listing',
+  'AP-AR Listing Report': 'ap-listing',
```
Comment (samplackett) [inline app/constants/report-types.js:5]: pedantic but probably just make this 'listing' or 'ap-ar-listing'

File: app/views/reports-list/ap-listing.njk [hunk 2]
```diff
@@ -11,21 +12,39 @@
       text: "Report List",
       href: "/report-list"
     },{
-      text: "AP Listing Report",
+      text: "AP/AR Listing Report",
       href: "/report-list/ap-listing"
```
Comment (samplackett) [inline app/views/reports-list/ap-listing.njk:16]: likewise I'd mirror this href in line with the generic term we use in report types

Comment (JungaBunga) [reply app/views/reports-list/ap-listing.njk]: changed naming to be ap-ar-listing

**Review by samplackett — APPROVED** [review]
that-is-bloody-lovely-blue-van-man

**Pull request merged**